File: 1.two-sums/two-sums.py
# /**
#  * 1. Two Sum
#  * @param {number[]} numbs
#  * @param {number} target
#  * @return {number[]}
#  */
def twoSum(numbs, target):
    indices = []
    for i in range(len(numbs)):
        for j in range(len(numbs)):
            if(not(i == j)):
                if(numbs[i] + numbs[j] == target):
                    indices = [j, i]
    return indices

File: problems/two-sums/two-sums.js
/**
 * 1. Two Sum
 * @param {number[]} numbs
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (numbs, target) {
	let indices = [];
	numbs.forEach((num, index) => {
		for (let i = 0; i < numbs.length; i++) {
			if (!(i === index)) {
				if (num + numbs[i] === target) {
					indices = [i, index];
				}
			}
		}
	});
	return indices;
};
